map_config/default_args.json

```diff
@@ -1 +1,22 @@
-{"worldforge": ["./map_config/worldforge.rb", "./simulation_ws/src/aws_robomaker_worldforge_worlds/worlds"], "no_roof_small_warehouse": ["./map_config/no_roof_small_warehouse.rb", "aws-robomaker-small-warehouse-world"], "bookstore": ["./map_config/bookstore.rb", "aws-robomaker-bookstore-world"], "small_warehouse": ["./map_config/small_warehouse.rb", "aws-robomaker-small-warehouse-world"], "small_house": ["./map_config/small_house.rb", "aws-robomaker-small-house-world"]}
\ No newline at end of file
+{
+    "no_roof_small_warehouse": [
+        "./map_config/no_roof_small_warehouse.rb",
+        "aws-robomaker-small-warehouse-world"
+    ],
+    "bookstore": [
+        "./map_config/bookstore.rb",
+        "aws-robomaker-bookstore-world"
+    ],
+    "small_warehouse": [
+        "./map_config/small_warehouse.rb",
+        "aws-robomaker-small-warehouse-world"
+    ],
+    "small_house": [
+        "./map_config/small_house.rb",
+        "aws-robomaker-small-house-world"
+    ],
+    "worldforge": [
+        "./map_config/worldforge.rb",
+        "./simulation_ws/src/aws_robomaker_worldforge_pkgs/aws_robomaker_worldforge_worlds/worlds"
```

Comment:
Why does this config have `simulation_ws/src` prefix and the others don't?

Reply:
the python script that loads the .rb file handles aws-robotics worlds and WorldForge world differently, see
https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch/blob/anson/worldforge-navigation/scripts/add_map_plugin.py#L36